lib/include/newWeightUpdateModels.h

```diff
@@ -158,6 +158,32 @@ class StaticPulse : public Base
         "$(updatelinsyn);\n");
 };
 
+//----------------------------------------------------------------------------
+// WeightUpdateModels::StaticPulseDendriticDelay
+//----------------------------------------------------------------------------
+//! Pulse-coupled, static synapse with heterogenous dendritic delays
+/*! No learning rule is applied to the synapse and for each pre-synaptic spikes,
+    the synaptic conductances are simply added to the postsynaptic input variable.
+    The model has 2 variables:
+    - g - conductance of scalar type
+    - d - dendritic delay in timesteps
+    and no other parameters.
+
+    \c sim code is:
+
+    \code
+    " $(addToDenDelay, $(g), $(d));\n\
+    \endcode*/
+class StaticPulseDendriticDelay : public Base
+{
+public:
+    DECLARE_MODEL(StaticPulseDendriticDelay, 0, 2);
+
+    SET_VARS({{"g", "scalar"},{"d", "uint8_t"}});
+
+    SET_SIM_CODE("$(addToDenDelay, $(g), $(d));\n");
```

Comment:
Hum ... this is the main naming/syntax question I suppose.
It resonates with the ```$(addtoinSyn) = ...``` and ```$(updatelinsyn)```, but
a) should it be called ```$(addtoinSynDelay, x, y)``` ?
b) should we finally also provide the syntax ```$(addtoinSyn, x)```?